data/taylorjeandev.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "Taylor Jean",
+  "url": "www.github.com/taylorjeandev",
+  "image": "",
+  "why": "To join a community of like minded people (developers, and fitness nerds)",
+  "weekly": ['Ground Control Daily', 'Bazooka Daily', 'Shadowboxing Daily']
```

Comment:
Please use double quotes instead of single.

Reply:
thank you for the code review and suggestions, aparna